docs/02-app/01-building-your-application/06-optimizing/10-third-parties.mdx

```diff
@@ -0,0 +1,135 @@
+---
+title: Third Parties
```

Comment:
@janicklas-ralph Can you update the examples (or create new ones) in the Next.js examples repo in this PR for each of the newer third-parties included?

We should have `with-google-maps` and `with-youtube-embed` updated (or created if they don't yet exist) that shows how to use these newer components.